lib/features/login/presentation/login_controller.dart

```diff
@@ -186,11 +186,13 @@ class LoginController extends GetxController {
     }
   }
 
-  void _getTokenOIDCSuccess(GetTokenOIDCSuccess success) {
+  void _getTokenOIDCSuccess(GetTokenOIDCSuccess success, OIDCConfiguration config) {
     log('LoginController::_getTokenOIDCSuccess(): ${success.tokenOIDC.toString()}');
     loginState.value = LoginState(Right(success));
     _dynamicUrlInterceptors.changeBaseUrl(kIsWeb ? AppConfig.baseUrl : _urlText);
     _authorizationInterceptors.setToken(success.tokenOIDC.toToken());
+    _authorizationInterceptors.setAuthenticationType(AuthenticationType.oidc);
+    _authorizationInterceptors.setOIDCConfiguration(config);
```

Comment:
why not set it at the same time, in one method?

Reply:
I wish it was clearer what we update.

Reply:
```dart
void setTokenAndAuthorityOidc({Token? newToken, OIDCConfiguration? newConfig}) {
    _token = newToken;
    _configOIDC = newConfig;
    _authenticationType = AuthenticationType.oidc;
  }

```